packages/playground/website/src/lib/state/url/resolve-blueprint-from-url.ts

```diff
@@ -121,6 +121,14 @@ export async function resolveBlueprintFromURL(url: URL) {
 
 	// Landing page
 	if (query.get('url')) {
+		/**
+		 * If the provided url has multiple query arguments, only the first one
+		 * is picked up as the url argument of the Playground website.
+		 *
+		 * @TODO This is standard browser behavior, but we should consider changing it
+		 * in the future to pick up all query arguments of the url if
+		 * they aren't valid Playground website query arguments.
```

Comment:
@adamziel in the case when a Playground URL looks like `https://playground.wordpress.net/?url=/wp-admin/post.php?post=1&action=edit`, the `action=edit` query argument won't be included in the `landingPage`.

It will work If the URL inside the argument is encoded, but not if it's a _plain_ URL like in this example.

This is expected browser behavior, but we could be smarter here and include all the query arguments after `url` that aren't valid Query API arguments.

Examples:

> https://playground.wordpress.net/?url=/wp-admin/post.php?post=1&action=edit

Would result in `/wp-admin/post.php?post=1&action=edit`


> https://playground.wordpress.net/?url=/wp-admin/post.php?post=1&action=edit&blueprint-url=http://test.com

Would result in `/wp-admin/post.php?post=1&action=edit`


> https://playground.wordpress.net/?blueprint-url=http://test.com&url=/wp-admin/post.php?post=1&action=edit

Would result in `/wp-admin/post.php?post=1&action=edit`

This could still lead to cases where the URL isn't fully read, but that would happen only if a valid QueryAPI argument is part of the `url` argument, but only in cases when the provided url isn't encoded.

Reply:
> This is expected browser behavior, but we could be smarter here and include all the query arguments after url that aren't valid Query API arguments.

I'd rather not, this would create a lot of confusion and inconsistencies in how the query API args are treated. There is an established and reliable way of escaping these values, let's stick to it.